ui/modal_templates.go

```diff
@@ -275,6 +296,12 @@ func (m *ModalTemplate) handle(th *decredmaterial.Theme, load *modalLoad) (templ
 		}
 		template = m.removeWallet(th)
 		return
+	case InfoTemplate:
+		if m.cancel.Button.Clicked() {
+			load.cancel.(func())()
+		}
+		template = m.verifyMessageInfo()
+		return
```

Comment:
This `InfoTemplate` is supposed to render for all the information on the pages i presume. currently, this renders only the `verifyMessageInfo`. We can either make this dynamic so it works with all info on the pages or have the `InfoTemplate` renamed to `verifyMsgInfoTemplate`.

Reply:
you're correct. My intention was to do that before pushing, I'll do that on my next commit.